impeller/aiks/experimental_canvas.cc

```diff
@@ -367,12 +356,33 @@ void ExperimentalCanvas::SaveLayer(
     return;
   }
 
+  std::shared_ptr<FilterContents> filter_contents;
+  if (paint.image_filter) {
+    filter_contents = paint.image_filter->GetFilterContents();
+  }
+
+  std::optional<Rect> maybe_subpass_coverage = ComputeSaveLayerCoverage(
+      bounds.value_or(Rect::MakeMaximum()),
+      transform_stack_.back().transform,  //
+      coverage_limit,                     //
+      filter_contents,                    //
+      /*flood_output_coverage=*/
+      Entity::IsBlendModeDestructive(paint.blend_mode),  //
```

Comment:
Should this also examine the paint.color_filter?

Reply:
We don't know that bit currently, but we will eventually: https://github.com/flutter/flutter/issues/154035

Reply:
(once we delete impeller::ColorFilter)